scripts/suse_build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+. $(dirname "$0")/suse_functions.sh
```

Comment:
I would recommend against finding the common functions this way.  See [here](http://mywiki.wooledge.org/BashFAQ/028).  I think using `BASH_SOURCE` would probably be best.